### ISSUE TITLE
assumption that students know setTimeout

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/mc-do-this-then-do-that#5]
This was something that made the stopwatch project during testing week really hard for some of us.

I'm not sure how you'd demonstrate async as simply as possible without `setTmeout`, but maybe provide an explanation in the readme, with the instructions #2 ?

**Comment by des-des** [comment]
can they google it? https://www.google.co.uk/search?q=setTimeout

**Comment by jsms90** [comment]
Yeah. They can. But understanding `setTimeout` doesn't really stop them from completing the exercise, so they won't necessarily pause the morning challenge for it.

Students just raised questions about it during solutions casting today.

**Comment by astroash** [comment]
Would it be possible to hide the ```setTimeout```'s in another file, then require them in like modules? This might make the code appear more like async code we are used to seeing.

**Comment by des-des** [comment]
@astroash @jsms90 sure we could hide the setTimeouts.

I actaully kinda originally tried to do this, wrapping all the exports in a function
```js
const asyncify = fn => (...args) => setTimeout(() => fn(...args), 200)
```
But then thought that was too complex.

We could have separate functions that were imported then used in the exercise. The one thing is that I want the async and sync solutions to be mirror images of each other, so we would want to change them also

**Comment by astroash** [comment]
@des-des do you happen to have a branch with this layout on? I'm happy to have a play with it with a pair of 'rookie eyes' to feedback whether it is actually more useful/ complicated

**Comment by des-des** [comment]
@astroash I am about tomorrow, ill put something together then we can talk about it!
